content/en/platform/corda/1.5/cenm/config-network-parameters.md

```diff
@@ -29,13 +29,12 @@ Ordered list of file paths to the node info files, or X500 names, of the notarie
 network.
 
 {{< note >}}
-Once a network has started, the only supported changes to notaries are to add new notaries at the end of the list or to remove existing ones as part of a decommissioning process.
+Once a network has started, the only supported changes to notaries are to add new notaries at the end of the list.
 {{< /note >}}
-Notaries can be removed from the list, in which case no new states can be created and reference it. The existing states which reference
-the removed Notary have to be moved to a new Notary before the pointed one is decommissioned. Notaries must be added
-to the end as Flows often use the ordering of notaries during selection (i.e. pick the first),
+
+Notaries must be added to the end, as flows often use the ordering of notaries during selection (i.e. pick the first),
```

Comment:
Should this sentence not be the same as the change made in other versions above?